src/main/java/com/redhat/cloud/policies/app/LightweightEngine.java

```diff
@@ -0,0 +1,43 @@
+package com.redhat.cloud.policies.app;
+
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+
+import javax.validation.constraints.NotNull;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.HeaderParam;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import java.util.Set;
+import java.util.UUID;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+import static javax.ws.rs.core.MediaType.TEXT_PLAIN;
+
+// TODO POL-649 Retries on all calls: https://quarkus.io/guides/smallrye-fault-tolerance
+// TODO POL-649 Exception mapper?
+@Path("/lightweight-engine")
+@RegisterRestClient(configKey = "engine")
+public interface LightweightEngine {
+
+    // TODO POL-649 The validation errors need to be properly forwarded to the browser.
+    /**
+     * Validates a condition with Hawkular.
```

Comment:
If you want, you could add an example of an condition. Maybe it's clear for everyone using this endpoint and hawkular besides me :D but I would not know how such a condition would look like at the moment

Reply:
Done.